### PR TITLE
Remove legacy extensions compilation from build

### PR DESCRIPTION
**Pull request by joaomoreno**
@jrieken The following changes did it:

- Update `vscode-nls-dev` to the latest version which contains the `NLSBundlePlugin`
- Change the `CopyWebpackPlugin` usage to not use `flatten` any more. This was screwing up with the `ignore` pattern.
- Fix extra files in shipped `html-language-features` extension. We have been shipping test fixtures from this extension, due to the issue related to `flatten` and `ignore`.
- Add webpack to the `python` extension. It simply didn't have any.

Using `diff`, I verified that all nls metadata files are in and all the hashes match :fireworks:! The only other differences are:

```
$ diff -rq * | grep -v nls.metadata
Only in extensions-original/html-language-features/server/dist: 19813-4spaces.html
Only in extensions-original/html-language-features/server/dist: 19813.html
Only in extensions-original/html-language-features/server/dist: 19813-tab.html
Only in extensions-original/html-language-features/server/dist: 21634.html
Only in extensions-original/html-language-features/server/dist: about.css
Only in extensions-original/html-language-features/server/dist: about.html
Only in extensions-original/html-language-features/server/dist: icon.pic
Only in extensions-original/html-language-features/server/dist: index.html
Only in extensions/python: dist
Only in extensions-original/python: out
Files extensions/python/package.json and extensions-original/python/package.json differ
```

### CODE REVIEW

File: extensions/shared.webpack.config.js [hunk 1]
```diff
@@ -9,11 +9,16 @@
...
 const path = require('path');
+const fs = require('fs');
 const merge = require('merge-options');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
-
+const { NLSBundlePlugin } = require('vscode-nls-dev/lib/webpack-bundler');
 
 module.exports = function withDefaults(/**@type WebpackConfig*/extConfig) {
+	const folderName = path.relative(__dirname, extConfig.context).split(/[\\\/]/)[0];
+	const pkgPath = path.join(__dirname, folderName, 'package.json');
```
Comment (jrieken) [inline extensions/shared.webpack.config.js]: Maybe simplify: `extConfig.context` is always the `__dirname` of the extension in question (not the dir of the shared config file) and I would have assumed that `join(extConfig.context, 'package.json')` does the job

Comment (joaomoreno) [reply extensions/shared.webpack.config.js]: Not in the cases that `extension.webpack.config.js` is deep in some `server` folder, unfortunately...

Comment (jrieken) [reply extensions/shared.webpack.config.js]: Oh, true. You need the outer most package.json... Bummer. Might deserve a doc comment then

File: extensions/shared.webpack.config.js [hunk 2]
```diff
@@ -9,11 +9,16 @@
...
 
 const path = require('path');
+const fs = require('fs');
 const merge = require('merge-options');
 const CopyWebpackPlugin = require('copy-webpack-plugin');
-
+const { NLSBundlePlugin } = require('vscode-nls-dev/lib/webpack-bundler');
 
 module.exports = function withDefaults(/**@type WebpackConfig*/extConfig) {
+	const folderName = path.relative(__dirname, extConfig.context).split(/[\\\/]/)[0];
```
Comment (joaomoreno) [inline extensions/shared.webpack.config.js]:
```suggestion
	// Need to find the top-most `package.json` file
	const folderName = path.relative(__dirname, extConfig.context).split(/[\\\/]/)[0];
```

File: extensions/shared.webpack.config.js (hunk 1 above)
Comment (joaomoreno) [reply extensions/shared.webpack.config.js]: Damn I just committed that directly in GitHub using the suggested changes feature. I'm living in the year 3000.

**Comment by joaomoreno** [comment]
Build looks great. Merging this.

**Pull request merged**